.github/workflows/code.yml

```diff
@@ -125,6 +125,7 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           context: .
           labels: ${{ steps.meta.outputs.labels }}
+          build-args: ${{ !(github.event_name == 'push' && startsWith(github.ref, 'refs/tags')) && 'SETUPTOOLS_SCM_PRETEND_VERSION=$(git describe --tags | cut -d - -f 1,2 --output-delimiter .dev)' }}
```

Comment:
We need a unique release number, so we could use seconds past epoch?
```suggestion
          build-args: ${{ startsWith(github.ref, 'refs/tags') && 'SETUPTOOLS_SCM_PRETEND_VERSION=$(date +%s)' }}
```

Reply:
Not that I understand the magic of GHA's domain specific expression language that drives this...

Reply:
Yeah, that'd do the trick

Reply:
> Not that I understand the magic of GHA's domain specific expression language that drives this...

It's a massive abuse of short circuiting, nothing "proper"